### DISCUSSION TITLE
Climate Autostart

### DISCUSSION THREAD

**Discussion by DerBo19** [Blackymas/NSPanel_HA_Blueprint#1986] [Ideas]
Hi everyone,

i´m using the climate function to heat up my house. In every room there is a nspanel wich give the temperature. Via the generic_thermostate the valve opens and closes. But everytime i restart my HA (e.g. when adding something to the configuration.yaml etc.) all heating functions on the panels are deactivated because a restart from ha restarts all the automations also. I´m wondering if it´s possible to add an "autostart" button to the blueprint so that after a restart of the blueprint the automatic-heating button on the panels (little flame on climate page) get activated and i dont have to go arround and activate on each panel one by one. OR EVEN BETTER: it should remind the "last state" and gets activated or not based on wether it was activated or not before restart. 
Also there are moments when my panels restart by themselves (sometimes at night for example). The rooms will get cold if i forgot to activate them by myself.

thanks for help

**Comment by edwardtfn** [comment]
Are you using the panel's relays to control your thermostats? If so, why don't you use the [add-on climate](https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/addon_climate.md)?

**Reply by DerBo19** [reply]
Hi edward,
im using the relays to control my shutter. You helped me a few month ago with this and it works perfectly.
I've been watching this for a while now. For the most part, the panels seem to retain their settings even after a restart. Maybe I was just unlucky. I'll have to keep watching this and see if the switching off the fire symbol by its own can be reproduced.

**Comment by 4irm** [comment]
> Hi everyone,
> 
> i´m using the climate function to heat up my house. In every room there is a nspanel wich give the temperature. Via the generic_thermostate the valve opens and closes. But everytime i restart my HA (e.g. when adding something to the configuration.yaml etc.) all heating functions on the panels are deactivated because a restart from ha restarts all the automations also. I´m wondering if it´s possible to add an "autostart" button to the blueprint so that after a restart of the blueprint the automatic-heating button on the panels (little flame on climate page) get activated and i dont have to go arround and activate on each panel one by one. OR EVEN BETTER: it should remind the "last state" and gets activated or not based on wether it was activated or not before restart. Also there are moments when my panels restart by themselves (sometimes at night for example). The rooms will get cold if i forgot to activate them by myself.
> 
> thanks for help

On my side everything is fine, at the very beginning I had to activate the heating functionality once, yet, even after a restart of HA the state remains to be as it should.

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/28482836/e3edb3d0-d1b8-4783-94ef-a3f26847ad58)
